account_ux/views/account_invoice_views.xml

```diff
@@ -30,6 +30,9 @@
             <field name="currency_id" position="attributes">
                 <attribute name="readonly">True</attribute>
             </field>
+            <field name="tax_line_ids" position="after">
+                <button string="_Add_Tax" states="draft" class="oe_link" type="action" name="%(action_account_invoice_tax_wizard_form)d"/>
+            </field>
         </field>
```

Comment:
@zaoral Yes, it was done on purpose, because odoo already incorporates it into the code.
https://github.com/odoo/odoo/blob/11.0/addons/account/views/account_invoice_view.xml#L489

Reply:
great